content/testing/fluent-assertions.md

```diff
@@ -0,0 +1,96 @@
+---
+title: FluentAssertions
+showMetadata: true
+editable: true
+showToc: true
+---
+
+สมมติว่าเรามี Data Structure ดังนี้ คำถามคือเราจะ Assert อย่างไร
+
+```c#
+public class Todo
+{
+    public int id  { get; set; }
+    public string title  { get; set; }
+    public bool completed  { get; set; }
+}
+
+public class TodoList
+{
+
+    public IEnumerable<Todo> GetTodoList()
+    {
+        return new Todo[]
```

Comment:
Very minor, you can omit type when defining array with initialized value.
Therefore, you can do.
```
            var expected =  new []
```

Reply:
Ok